client/src/pages/ProfileListing/CoffeCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import { IconButton, CardMedia } from '@material-ui/core';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+import Rating from '@mui/material/Rating';
+
+const CoffeCard = (props) => {
+  const { avatarUrl, title, description, imageUrl, maintitle, price, country } = props;
+  return (
+    <>
+      <Card>
+        <div style={{ display: 'flex', justifyContent: 'center', marginTop: '20px' }}>

```

Comment:
Let's use `Box` component instead of divs. And instead of the style property, use the `sx` property.